docs/en/operation/Basic-Logging.md

```diff
@@ -262,3 +269,26 @@ FREE_BYTES, CAPACITY_BYTES]) returned BlockMasterInfo{capacityBytes=11453246122,
 capacityBytesOnTiers={}, freeBytes=11453237930, liveWorkerNum=0, lostWorkerNum=0, usedBytes=8192, usedBytesOnTiers={}} in 600 ms
 2020-03-08 23:40:44,374 WARN  AlluxioFuseFileSystem - statfs(path=/) returned 0 in 1200 ms
 ```
+
+### Redirecting debug log for certain classes
+
+Sometimes it is useful to separate the log for certain classes to a separate log.
+This can be useful for reasons including but not limited to:
+1. Clearly separate the wanted logs for further analysis.
+1. Avoid `master.log` or `worker.log` being too big or have too many files created by log rotation.
+1. Use a separate logger to send logs to a remote endpoint like a socket.
+
+This can be achieved by adding a separate logger in the `conf/log4j.properties`.
+For example, the below example redirects debug logs of `alluxio.master.StateLockManager` to a separate set of files,
+so the `master.log` will not be full of the DEBUG logs created by `alluxio.master.StateLockManager`.
+
+```properties
+log4j.category.alluxio.master.StateLockManager=DEBUG, slm
```

Comment:
what's the meaning of slm?

Reply:
`slm` just means `StateLockManager`, do you have a better name in mind? @LuQQiu

Reply:
updated